subsys/bluetooth/mesh/Kconfig

```diff
@@ -93,68 +93,95 @@ endif # BT_MESH_RELAY
 
 endif
 
-config BT_MESH_NLC_PERF_CONF
-	bool "Enable NLC performance configurations"
+config BT_MESH_NLC_PERF_BASELINE
+	bool
 	select BT_MESH_COMP_PAGE_2
 	select BT_MESH_COMP_PAGE_1
 	help
-	  Enables performance configurations required for NLC profiles.
-	  When implementing the NLC Basic Lightness Controller profile
-	  individually or in conjunction with other NLC profiles, also
-	  enable CONFIG_BT_MESH_NLC_PERF_LIGHTNESS_CTRL.
-
-choice BT_MESH_NLC_PERF_OPTIONS
-	prompt "NLC configuration options"
-	depends on BT_MESH_NLC_PERF_CONF
+	  Provides baseline performance requirements for NLC profiles.
```

Comment:
Wait, what's the idea with a `help` section for a promptless option? This option (and the `STRICT` one) will not be possible to select directly by users. If this is meant for readers of this file, I think it's better to use a comment? (Also, I'm not 100% sure whether adding this `help` actually makes the option user-accessible, if it does then we definitely don't want it)